dev/integration_tests/simple_scroll/test/frame_policy.dart

```diff
@@ -0,0 +1,105 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
```

Comment:
To reduce the number of new files introduced, can we put this under `macrobenchmarks/test`, and create a new page in `macrobenchmarks` for the purpose of `simple_scroll`?

Reply:
I don't feel comfortable doing that because it's technically not a performance benchmark but more like a functioning test. But if you insist I'll make the change.

Reply:
The reporting of metrics like `average_delay_fullyLive_millis` and `average_delay_benchmarkLive_millis` makes me feel that this is more or less a performance test. It's testing the performance of the input events simulation which justifies `benchmarkLive` (i.e., everything would still run if we don't have `benchmarkLive`, but the simulation performance would be bad and inaccurate). Such performance metrics also prevent future accidents that suddenly make our simulation much less accurate.

Reply:
OK. Then I'll wait this one until that renaming PR lands to avoid conflicts.

Reply:
Done.